os-audit/cos-auditd-logging.yaml

```diff
@@ -155,8 +155,6 @@ data:
         Systemd_Filter  SYSLOG_IDENTIFIER=audit
         Path            /var/log/journal
         DB              /var/lib/cos-auditd-fluent-bit/pos-files/audit.db
-        Buffer_Max_Size 20MB
```

Comment:
can you add description for this change into PR description

Reply:
Done